base-server/index.test.ts

```diff
@@ -927,9 +928,9 @@ it('subscribes clients', async () => {
   })
   expect(test.reports[3][1].meta.clients).toEqual(['10:a'])
   expect(test.reports[3][1].meta.status).toEqual('processed')
-  expect(test.app.subscribers).toEqual({
+  expect(test.app.subscribers).toMatchObject({
```

Comment:
`toMatchObject` is required to templates like `expect.stringMatching(/white|yellow/)`

Let’s use `toEqual` if we do not use templates.

Reply:
Mainly yes but also may be useful to keep only meaningful fields enumerated in test.

Fixed for subscribers (really redundant since I stopped using null-safe `noop` function as a `subscriber.unsubscribe` default)